testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -706,11 +710,30 @@ pub fn next_block_and_mine_commit(
         timeout_secs,
         &[coord_channels],
         &[commits_submitted],
+        true,
     )
 }
 
 /// Mine a bitcoin block, and wait until:
-///  (1) a new block has been processed by the coordinator
+///  (1) 2 block commits have been issued ** or ** more than 10 seconds have
+///      passed since (1) occurred
```

Comment:
This is copy-pasted from these other methods, but I think it should say:
```suggestion
/// Mine a bitcoin block, and wait until a block commit has been issued
/// ** or ** more than timeout_secs seconds have passed
```